service-example/README.md

```diff
@@ -0,0 +1,44 @@
+# Service model example
+
+A service model runs beside other models. 
+Typically this can be database which stores information as a workflow progresses. This example contains two models the service and a model which communicates with it, which we have called the client.
```

Comment:
```suggestion
Typically this can be a database which stores information as a workflow progresses. This example contains two models the service and a model which communicates with it, which we have called the client.
```